java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -243,4 +243,30 @@ public interface StringBaseCommands {
      * }</pre>
      */
     CompletableFuture<String> getrange(String key, int start, int end);
+
+    /**
+     * Appends a value to a key. If `key` does not exist it is created and set as an empty string, so
+     * `APPEND` will be similar to `SET` in this special case.
```

Comment:
Not sure that we need such detailed doc, but if you refer to `set`, add a link to that command. Javadoc renderer will make it clickable.
```suggestion
     * Appends a value to a key. If `key` does not exist it is created and set as an empty string, so
     * `APPEND` will be similar to {@see #set} in this special case.
```
(I hope I didn't mistake in the syntax)

Reply:
got it

Reply:
```
     * Appends <code>value</code> to the string at <code>key</code>. If <code>key</code> does not exist it is created and set as an empty string, so
     * `APPEND` will be similar to `SET` in this special case.
```
Use code tags when referring to arguments and Java types.